### DISCUSSION TITLE
Anyone else troubling starting web flutter builds hosted on github ?

### DISCUSSION THREAD

**Discussion by RoarGronmo** [codemagic-ci-cd/codemagic-docs#2581] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

-

### Steps to reproduce

Trying to build my projects get this result duiring preparing for build:
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/11217296/8e4df5ed-34fb-49a7-9a3c-6564a136b491)

Was working 30 mins ago...

### Expected results

Start build

### Actual results

Nothing

### Build id (optional)

66606519e9a50f54f55b5ccc

**Comment by mikhail-tokarev** [comment]
hey 
please check this thread https://github.com/orgs/codemagic-ci-cd/discussions/2568 or status page

**Comment by helinanever** [comment]
Hi @RoarGronmo , this is a known issue affecting macOS machines right now, see this discussion: https://github.com/orgs/codemagic-ci-cd/discussions/2568
